Packages/io.chainsafe.web3-unity.lootboxes/package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "io.chainsafe.web3-unity.lootboxes",
+    "version": "1.0.0",
+    "displayName": "web3.unity SDK Lootboxes",
+    "description": "This additional package provides you with sample scene and sample scripts for opening lootboxes of various types",
+    "license": "LGPL-3.0-only",
+    "licensesUrl": "https://github.com/ChainSafe/web3.unity/blob/main/LICENSE",
+    "documentationUrl": "https://docs.gaming.chainsafe.io/",
+    "dependencies": {
+        "io.chainsafe.web3-unity": "2.5.0"
+    },
+    "keywords": [
+        "web3",
+        "ethereum",
+        "evm",
+        "blockchain",
+        "nft"
+    ],
+    "author": {
```

Comment:
Samples should be added here or it won't be importable from the package manager, take a look [here](https://docs.unity3d.com/Manual/cus-samples.html) for examples on how to do that, you can also look at web3.unity's `package.json`

Reply:
ooooh, thanks for that!